pkg/execution/batch/redis.go

```diff
@@ -133,14 +136,28 @@ func (b redisBatchManager) Append(ctx context.Context, bi BatchItem, fn inngest.
 	return result, nil
 }
 
+func (b redisBatchManager) isSharded(ctx context.Context, batchPointer string, functionId uuid.UUID) (redis_state.RetriableClient, bool) {
```

Comment:
We can remove this function 60s after rolling out, when all batches are sharded.